databases/src/create/PostgresAccountWizard/createPostgresServer/PostgresServerSkuStep.ts

```diff
@@ -0,0 +1,251 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { AzureWizardPromptStep, IAzureQuickPickItem } from "vscode-azureextensionui";
+import { openUrl } from "../../../utils/openUrl";
+import { localize } from "../../../utils/localize";
+import { nonNullProp } from "../../../utils/nonNull";
+import { AbstractSku, PostgresServerType } from "../abstract/models";
+import { IPostgresServerWizardContext } from "../IPostgresServerWizardContext";
+
+interface ISkuOption {
+    label: string;
+    description: string;
+    sku: AbstractSku;
+    group?: string;
+}
+
+export class PostgresServerSkuStep extends AzureWizardPromptStep<IPostgresServerWizardContext> {
+    public async prompt(
+        context: IPostgresServerWizardContext
+    ): Promise<void> {
+        const placeHolder: string = localize(
+            "selectPostgresSku",
+            "Select the Postgres SKU and options."
+        );
+        const pricingTiers: IAzureQuickPickItem<AbstractSku | undefined>[] = await this.getPicks(nonNullProp(context, 'serverType'));
+        pricingTiers.push({
+            label: localize('ShowPricingCalculator', '$(link-external) Show pricing information...'),
+            onPicked: async () => {
+                await openUrl('https://aka.ms/AAcxhvm');
+            }, data: undefined
+        });
+
+        context.sku = (await context.ui.showQuickPick(pricingTiers, { placeHolder, suppressPersistence: true, enableGrouping: true })).data;
+    }
+
+    public shouldPrompt(context: IPostgresServerWizardContext): boolean {
+        return context.sku === undefined;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/require-await
+    public async getPicks(serverType: PostgresServerType): Promise<IAzureQuickPickItem<AbstractSku | undefined>[]> {
+        const options: IAzureQuickPickItem<AbstractSku | undefined>[] = [];
+        const skuOptions: ISkuOption[] = serverType === PostgresServerType.Single ? singleServerSkus : flexibleServerSkus;
+
+        skuOptions.forEach((option) => {
+            options.push({
+                label: option.label,
+                description: localize(
+                    nonNullProp(option.sku, "name"),
+                    option.description
+                ),
+                data: option.sku,
+                group: option.group || localize('addlOptions', 'Additional Options')
+            });
+        });
+        return options;
+    }
+}
+
+const recommendedGroup = localize('recommendGroup', 'Recommended');
+const singleServerSkus: ISkuOption[] = [
```

Comment:
Can these be obtained by an API? I can see them being a frequent maintenance item.

Reply:
Tracking all suggestions for updating "create"/"utils" folder here: #1013